controllers/homeController.js

```diff
@@ -1,33 +1,32 @@
 const { categories } = require('../categories');
+const GreenPickApp = require("../models/greenPickApp");
 
 /**
  * Render the index.ejs file (index page).
  */
-exports.getIndex = (req, res) => {
-  // to focus on one category, set activeCategory to the active label, else set to NULL
-  let activeCategory = null;
+module.exports = {
+  /**
+   * Get all app from data base
+   * TODO: limit the apps show on index page
+   */
+  getAllApps: (req, res, next) => {
+    GreenPickApp.find((error, apps) => {
+      if(error) {
+        res.render("error");
```

Comment:
instead of simply rendering this, please require errorController and call respondNoResourceFound (or something) ^^ not super important but best practice